examples/restricted-assets/scripts/1-assetConfig.js

```diff
@@ -22,21 +22,14 @@ async function run (runtimeEnv, deployer) {
   const escrowLsig = await deployer.loadLogic('clawback-escrow.py', [], escrowParams);
   const escrowAddress = escrowLsig.address();
 
-  const assetModFields = {
-    manager: creator.addr,
-    reserve: creator.addr,
-    freeze: creator.addr,
-    clawback: escrowAddress
-  };
-
   /** Update clawback address to escrow **/
   console.log('* Updating asset clawback to escrow *');
   const assetConfigParams = {
     type: types.TransactionType.ModifyAsset,
     sign: types.SignType.SecretKey,
     fromAccount: creator,
     assetID: assetInfo.assetIndex,
-    fields: assetModFields,
+    fields: { clawback: escrowAddress }, // only pass the field you want to update
```

Comment:
Instead of removing `assetModFields`, can we change it to the hardcoded object here.

Reply:
the idea is that we update only the `clawback` address, all rest stays as it's defined in`asa.yaml`